packages/ember-glimmer/lib/component-managers/curly.js

```diff
@@ -307,13 +307,13 @@ export default class CurlyComponentManager extends AbstractManager {
     let { component, args, argsRevision, environment } = bucket;
 
     if (DEBUG) {
-       this._pushToDebugStack(component._debugContainerKey, environment)
+      this._pushToDebugStack(component._debugContainerKey, environment)
     }
 
     bucket.finalizer = _instrumentStart('render.component', rerenderInstrumentDetails, component);
 
     if (!args.tag.validate(argsRevision)) {
-      let props = processComponentArgs(args);
+      let props = processComponentArgs(args, component._debugContainerKey.split(':')[1]);
```

Comment:
This `.split(':')` is only needed for debug builds, can you restructure things to avoid doing the extra work in production?

Reply:
does `DEBUG` work inline ? 
can I do this `DEBUG ? component._debugContainerKey.split(':')[1] : '' ` ?
or there is better way

Reply:
It does work in line.  `DEBUG` is replaced at build time with the literal `true` or `false` (depending on build type).  If we use a ternary here, please extract to separate variable declaration though...